mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -1,3 +1,24 @@
 .TYPE-reference .RIGHT-Xsubject.card-editor {
   display: none
-}
\ No newline at end of file
+}
+
+.modal-header {
+  display: none;
+}
+
+.link-to-guide {
+  font-size: small;
+  margin-left: auto!important;
+  text-decoration: underline;
+}
+
+.custom-text {
+  font-weight: bold;
+}
+
+.alert_message {
+  align-self: center;
+  flex-direction: column;
+  margin-right: auto;
+  padding-left: 0.5rem;
+}
```

Comment:
also a little too generic. "attribution-alert-message" perhaps?  (hyphens are a bit more idiomatic in CSS and underscores are in ruby)

I would especially recommend using bootstrap for padding, because those classes are responsive. Eg `ps-2` will use a little less space in mobile.